android/src/main/cpp/NativeProxy.cpp

```diff
@@ -348,19 +349,41 @@ void NativeProxy::installJSIBindings(
 
   std::weak_ptr<ErrorHandler> weakErrorHandler = errorHandler;
 
+  auto convertToArray = [](jsi::Runtime &rt, const std::string &value) {
+    std::vector<float> transformMatrixList;
+    std::istringstream stringStream(value);
+    std::copy(
+        std::istream_iterator<float>(stringStream),
+        std::istream_iterator<float>(),
+        std::back_inserter(transformMatrixList));
```

Comment:
Can you please explain what exactly happens here? 😄

Reply:
It reads from the `value` string till the end, parsing it into floats, and pushing those floats to the `transformMatrixList`.

> `std::istream_iterator` is a single-pass input iterator that reads successive objects of type T from the [`std::basic_istream`](https://en.cppreference.com/w/cpp/io/basic_istream) object for which it was constructed, by calling the appropriate `operator>>`

from `std::istream_iterator` docs